packages/flutter/test/material/bottom_navigation_bar_test.dart

```diff
@@ -39,7 +39,7 @@ void main() {
       ),
     );
 
-    await tester.tap(find.text('Alarm'));
+    await tester.tap(find.text('Alarm'), warnIfMissed: false);
```

Comment:
Why is this now necessary? Seems odd...

Reply:
The text is not hittestable itself but the its ancestor (the nav bar item) is. It seems when a bottom nav bar item is pressed other items in the same bottom nav bar fade out temporarily.